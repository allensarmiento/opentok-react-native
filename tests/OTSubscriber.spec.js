import React from 'react';
import renderer from 'react-test-renderer';

import OTSubscriber from '../src/OTSubscriber';

jest.mock('../src/OT', () => ({
  nativeEvents: jest.fn()
}));

describe('OTSubscriber', () => {
  describe('no props', () => {
    it('should render an empty view', () => {
      const subscriber = renderer.create(<OTSubscriber />).toJSON();
      expect(subscriber).toMatchSnapshot();
    });
  });

  describe('with props', () => {

  });
});