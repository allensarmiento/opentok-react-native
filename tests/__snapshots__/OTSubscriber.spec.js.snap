// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OTSubscriber no props should render an empty view 1`] = `
<View
  style={Object {}}
/>
`;
